arrows/ocv/resection_camera.cxx

```diff
@@ -74,22 +105,24 @@ ::set_configuration( vital::config_block_sptr config )
                                                      d_->reproj_accuracy );
   d_->max_iterations = config->get_value< int >( "max_iterations",
                                                  d_->max_iterations );
+  std::stringstream ss( config->get_value< std::string > ( "focal_scales", "1" ) );
+  d_->focal_scales.clear();
+  ss >> d_->focal_scales;
 }
 
 // ----------------------------------------------------------------------------
 bool
 resection_camera
 ::check_configuration( vital::config_block_sptr config ) const
 {
-  auto good_conf = true;
```

Comment:
In `check_configuration` it is good practice to do all the checks before returning false if one failed.  The reason for this is that a config file may have multiple errors and we want all of those errors logged before failing.  Otherwise, the user will just see the first error, fix it, run again, and then see the second error, fix it, run again, etc.  It can take many more iterations to find all the problems.  It's better just to report all the errors you can find.